src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -43,6 +45,119 @@ public String getSummary() {
         }
     }
 
+    /**
+     * Information about a package as returned by pkg.info_installed and
+     * pkg.info_available
+     */
+    public static class Info {
```

Comment:
The idea here is similar to what we have with `Key` that the module name is part of the path you specify when referring to the type.

```java
import com.suse.saltstack.netapi.calls.modules.Pkg;

Pkg.Info a = ...
```